subsys/bluetooth/host/iso.c

```diff
@@ -414,7 +419,12 @@ static struct net_buf *hci_le_set_cig_params(struct bt_iso_create_param *param)
 		sys_put_le24(param->chans[0]->qos->rx->interval, req->s_interval);
 		req->s_latency = sys_cpu_to_le16(param->chans[0]->qos->rx->latency);
 	} else {
-		/* Use TX values if RX is disabled */
+		/* Use TX values if RX is disabled
+		 * If TX is disabled then the values don't have any meaning, as
```

Comment:
I guess it should say "If RX is disabled then..." here?

Reply:
Too much copy paste... :)